### ISSUE TITLE
Autofix Issue

### ISSUE THREAD

**Issue by SJLennon** [halcyon-tech/vscode-rpgle#161]
Been out of the loop for a while and just getting back, so maybe I'm missing something...

I found that the "Fix all auto-fixable Problems" has  vanished.  But per issue #148 it is now right click and "Format Document". So it still largely works.

Following snippet does not correct correctly.

Before:
![image](https://user-images.githubusercontent.com/67484051/200183427-28e23005-aeed-4fbb-9a58-85cef496fcb4.png)

After Format Document, there are missing opening brackets:
![image](https://user-images.githubusercontent.com/67484051/200183499-76f2e884-df21-48ad-9a78-dd6cb3d0bd25.png)

Running Format Document again just adds another closing bracket.

This is the unfixed code:
```
**Free        
        SELECT;
          WHEN SQLSTT = SQLSuccess;
            EofIn = *off;
          WHEN SQLSTT = SQLNoMoreData;
            EofIn = *ON;
          OTHER;
            SQLProblem('00': 0: 'Fetch EC_In into :IPRUCALCP');
        ENDSL;

```

**Comment by worksofliam** [comment]
Oh damn. Someone did actually mention about the bracket issue. I will fix that today.

**Comment by SJLennon** [comment]
@worksofliam Another question...
Didn't it used to fix errors like this:
![image](https://user-images.githubusercontent.com/67484051/200191042-5f335d5f-f479-4636-b71d-41bf876e1b5f.png)
Or has my memory gone?

**Comment by worksofliam** [comment]
@SJLennon the autofix never fixes those qualified lint rules.

**Comment by worksofliam** [comment]
@SJLennon fix will be available in 0.15.6!
